docs/production/data-sync/git-integration.mdx

```diff
@@ -29,71 +30,68 @@ you don't have to enter your SSH key passphrase every time you perform an action
 
 </Warning>
 <Steps>
-<Step title="Check preferences.yaml">
- Ensure that `.preferences.yaml` is added to your repo's `.gitignore` file so that your Git settings aren't pushed to Github, unless you want them to.
+<Step title="Check .gitignore settings">
+    Ensure that `.preferences.yaml` is added to your repo's `.gitignore` file so that your Git settings aren't pushed to Github.
 </Step>
-<Step title="Navigate to your Mage settings">
-    Wizard > Settings
-    <div align="center">
-    <img src="https://github.com/mage-ai/assets/blob/main/mage-settings.png?raw=True" width="50%"/>
-    </div>
+<Step title="Navigate to Git settings">
+    Navigate to the Wizard icon in the top right, then select _Settings_ to open your Mage settings.
+    <Frame>
+    <img src="https://github.com/mage-ai/assets/blob/main/mage-settings.png?raw=True"/>
+    </Frame>
+    There should be an option for _Git settings_ on the left side of the settings modal.
 </Step>
-<Step title="Navigate to the 'Git settings' section">
+<Step title="Select SSH as the authentication type.">
+    Change the authentication type to SSH. The interface will change for you to input a remote repo url. 
+    <Frame>
+    <img src="https://github.com/mage-ai/assets/blob/main/git-settings.png?raw=True"/>
+    </Frame>
 </Step>
 <Step title="Set up your SSH key">
-  This will allow your machine to access GitHub. 
-  Check out this sections this [guide](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) to set up your SSH key.
-</Step>
-<Step title="Select SSH as the authentication type.">
+  If you haven't already, configure your SSH key— this will allow your instance to access GitHub. You can click the link within the app to be taken to a page with instructions on how to set up your SSH key. Check out this sections this [guide](/development/git/generate-an-ssh-token) to set up your SSH key.
```

Comment:
the formatting on this doc is broken: https://mage-staging.mintlify.app/development/git/generate-an-ssh-token

Reply:
That's a mintlify bug that the team told us was a quick fix, but has not been resolved. I will remove the table